app/config/Switches.scala

```diff
@@ -70,6 +76,12 @@ class Switches(config: LoginConfig, s3Client: AmazonS3) extends Loggable {
       val source = Source.fromInputStream(result.getObjectContent).mkString
       val statesInS3 = Json.parse(source).as[Map[String, SwitchState]]
 
+      // we're refreshing every minute, only sendStillActiveNotification once an hour
+      val isOnTheHour = Try(DateTime.now().getMinuteOfHour % 0 == 0).getOrElse(false)
```

Comment:
This might bury a problem. What exception are you guarding against here?

Reply:
a `java.lang.ArithmeticException` can be thrown with `DateTime.now().getMinuteOfHour % 0`

Reply:
That's because I'm a muppet. You can't `% 0`. This can just be `getMinuteOfHour == 0`. 🤦‍♂

Reply:
Or perhaps it is worth making a one hour scheduler that is less of a hack....